src/js/control-bar/playback-rate-menu-button.js

```diff
@@ -98,7 +98,7 @@ PlaybackRateMenuButton.prototype.updateVisibility = function(){
  */
 PlaybackRateMenuButton.prototype.updateLabel = function(){
   if (this.playbackRateSupported()) {
-    this.labelEl_.innerHTML = this.player().playbackRate() + 'x';
+    this.labelEl_.innerHTML = `${this.player().playbackRate()}x`;
```

Comment:
I don't think string templates were meant to replace every instance of string concatenation everywhere. They work beautifully in the examples like `some text ${var} some text`. i.e. "I need to drop a variable in the middle of and existing string of text". And multi-line strings.

Other cases are "I need to append text to a an existing value". `width + 'px'`. That's not 'modifying a template', that's creating a value.

Maybe put more succinctly, in one you're adding a value to a string, in the other you're adding a string to a value. So we could use that for the rule around consistency.

In this specific example, `${this.player().playbackRate()}x`, the most important thing I need to see there is that the `x` is getting appended to the value, but visually that gets lost in the junkiness of curly braces and parens. Whereas with `this.player().playbackRate() + 'x'`, it's the first thing I see.